src/olympia/devhub/tests/test_views.py

```diff
@@ -2142,7 +2148,7 @@ def test_delete_button_disabled(self):
         assert "You cannot remove an individual file" in tip.attr('title')
 
     def test_delete_button_multiple(self):
-        file = self.addon.latest_version.files.all()[0]
+        file = self.addon.current_version.files.all()[0]
```

Comment:
It turns out that `find_latest_public_version()` and therefore `current_version` can return a non public version if the add-on itself is not public, by design. That sucks, but we'll deal with that later...